ksqldb-common/src/main/java/io/confluent/ksql/util/BytesUtils.java

```diff
@@ -65,6 +67,18 @@ private BytesUtils() {
       Encoding.BASE64, v -> base64Decoding(v)
   );
 
+  public static ByteOrder byteOrderType(final String byteOrderStr) {
+    if (byteOrderStr.equalsIgnoreCase(ByteOrder.BIG_ENDIAN.toString())) {
```

Comment:
If `byteOrderStr ` is null then `byteOrderStr.equalsIgnoreCase ` will throw an NPE

Reply:
Done